components/style/stylist.rs

```diff
@@ -294,7 +294,8 @@ impl Stylist {
                                          pseudo: &PseudoElement,
                                          parent: Option<&Arc<ComputedValues>>,
                                          default: &Arc<ComputedValues>,
-                                         inherit_all: bool)
+                                         inherit_all: bool,
```

Comment:
I think it's more elegant to receive CascadeFlags directly here, instead of multiple bool args, and the patch shouldn't become more complex, wdyt?